system/Autoloader/Autoloader.php

```diff
@@ -233,12 +245,21 @@ protected function loadInNamespace($class)
 			return false;
 		}
 
-		foreach ($this->prefixes as $namespace => $directory)
+		foreach ($this->prefixes as $namespace => $directories)
 		{
-			if (strpos($class, $namespace) === 0)
+			if (is_string($directories))
+			{
+				$directories = [$directories];
+			}
+			foreach ($directories as $directory)
 			{
-				$filePath = $directory.str_replace('\\', '/', substr($class, strlen($namespace))).'.php';
-				return $this->requireFile($filePath);
+				if (strpos($class, $namespace) === 0) {
+					$filePath = $directory . str_replace('\\', '/', substr($class, strlen($namespace))) . '.php';
+					$filename = $this->requireFile($filePath);
```

Comment:
We can have more than one directories for one namespace. So we can't `return` here if `$this->requireFile()` is `false`.